src/main/java/com/mongodb/spark/sql/connector/config/ReadConfig.java

```diff
@@ -191,6 +199,74 @@ public final class ReadConfig extends AbstractMongoConfig {
 
   private static final String STREAM_LOOKUP_FULL_DOCUMENT_DEFAULT = FullDocument.DEFAULT.getValue();
 
+  enum StreamingStartupMode {
+    /**
+     * Is equivalent to {@link #LATEST}, and is used to discriminate a situation when the default
+     * behavior is configured explicitly.
+     */
+    DEFAULT_INTERNAL,
+    /** @see #DEFAULT_INTERNAL */
+    LATEST,
+    TIMESTAMP;
+
+    static StreamingStartupMode fromString(final String userStartupMode) {
+      try {
+        return userStartupMode.equals(STREAMING_STARTUP_MODE_DEFAULT)
+            ? DEFAULT_INTERNAL
+            : StreamingStartupMode.valueOf(userStartupMode.toUpperCase());
+      } catch (IllegalArgumentException e) {
+        throw new ConfigException(format("'%s' is not a valid Startup mode", userStartupMode));
+      }
+    }
+  }
+
+  /**
+   * The start up behavior when there is no stored offset available.
+   *
+   * <p>Specifies how the connector should start up when there is no offset available.
+   *
+   * <p>Resuming a change stream requires a resume token, which the connector stores as / reads from
+   * the offset. If no offset is available, the connector may either ignore all/some existing source
+   * data, or may at first copy all existing source data and then continue with processing new data.
```

Comment:
The Kafka Connector can do this, but there is no `StreamingStartupMode.COPY_EXISTING` in the Spark Connector. We should say something like the following:

```suggestion
   * the offset. If no offset is available, the connector may either ignore all existing data, or may read an offset from the configuration.
```